tests/periph/uart_locate_pins/Makefile

```diff
@@ -0,0 +1,21 @@
+BOARD ?= nucleo-f767zi
+
+# This needs to be here so that this app's `soft_uart_params.h` is found used
+# of the one in $(RIOTBASE)/drivers/soft_uart/include.
+INCLUDES += -I$(abspath $(CURDIR))/include
+
+include ../Makefile.periph_common
+
+USEMODULE += soft_uart
+USEMODULE += stdio_null
+
+include $(RIOTBASE)/Makefile.include
+
+ifneq ($(MCU),esp32)
+  # We only need 1 thread (+ the Idle thread on some platforms) and we really
+  # want this app working on all boards.
```

Comment:
I guess you could even do

```
DISBALE_MODULE += core_thread
```

Mind you that esp32 is not the only platform that uses `core_idle_thread`

Reply:
That's why it is 2 (because of the idle thread) or 3 on ESP (because of some ESP32 specific worker thread)

Reply:
ah ok - is that also true for esp8266?

Reply:
I think not, but then again I'm not sure. But `DISABLE_MODULE += core_thread` works even better anyway.